Sprint-3/revise/implement/count.test.js

```diff
@@ -15,3 +15,96 @@
 // And a character char that does not exist within the case-sensitive str,
 // When the function is called with these inputs,
 // Then it should return 0, indicating that no occurrences of the char were found in the case-sensitive str.
+//---------------------------------------------------------------------------------------------------
+
+//// Converts a number into its English word representation
+function numberToWords(num) {
```

Comment:
I think you can safely delete all of this

Reply:
Oops, you're right! 🙈 I've deleted the lines as suggested